website/content/docs/blueprint-reference/components.md

```diff
@@ -128,4 +130,99 @@ spec:
         enabled: true
         manifest:
           url: "https://raw.githubusercontent.com/metallb/metallb/v0.14.3/config/manifests/metallb-native.yaml"
-```
\ No newline at end of file
+```
+
+### Kustomize Manifest based addons
+
+The users can now apply customizations on top of the url based manifests. They can specify `inline patches` and keyword-based `images` kustomization in the blueprint. The boundless controller applies these kustomize transformations on the manifest objects and deploys them in the cluster.
```

Comment:
**suggestion**: To provide a bit more context on what Kustomize options are supported, may be we can add the following here:

---
The users can now customize the static resources for URL based Manifest addons. These customization are based on [Kustomize](https://kustomize.io/).

Following `kustomize` primitives are currently supported:
* [Built-in images transformer](https://github.com/MirantisContainers/boundless-cli/blob/v0.2.0/images).
* [Inline patches](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/)